go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/FZambia/statik v0.1.2-0.20180217151304-b9f012bb2a1b
 	github.com/FZambia/viper-lite v0.0.0-20171108064948-d5a31e6aa18b
 	github.com/ajstarks/svgo v0.0.0-20180226025133-644b8db467af
-	github.com/centrifugal/centrifuge v0.8.1-0.20200510075015-d24832cd40bb
+	github.com/centrifugal/centrifuge v0.8.2
```

Comment:
`go.sum` should change too due to this update, also Centrifugo uses vendoring so `go mod tidy && go mod vendor` had to be called. Actually I can do everything myself when will prepare new release - including changelog